testings/test_open.py

```diff
@@ -200,6 +209,20 @@ def test_multiple_open(self):
 except ImportError:
     pass
 
+@pytest.mark.parametrize('spool_dir', spool_file_dirs)
+def test_np_spool_open(spool_dir):
+    with pytest.raises(ValueError) as e_info:
+        data, info = sif_parser.np_spool_open(spool_dir, ignore_missing=False)
```

Comment:
Do you have the result loaded and generated by AndorSolis? If we could compare numbers from the official conversion, it would be more robust.